docs/appendix/troubleshooting.md

```diff
@@ -239,7 +115,5 @@ None at this time.
 
 ### Browser Plugins
 
-1. **PLUGIN:** "**Allow right click - simple copy**" - This Chrome plugin prevents the user from configuring a Service in the Viam app (**app.viam.com** > **Config** > **3 Services**).
-
-   **Indication:** User unable to select a Service **Type** in the **Create Service** Pane.
-After clicking to display the drop-down list, choosing an item does not populate the drop-down's "Selection."
+- **Chrome Plugin: Allow right click** - This Chrome plugin interferes with the [Viam app](https://app.viam.com)'s ability to configure a Service.
```

Comment:
```suggestion
- **Chrome Plugin: Allow right click** - This Chrome plugin interferes with the [Viam app](https://app.viam.com)'s ability to configure a service.
```
I *think* it should be lower case here since it's not talking about any specific service?